File: Python_Prototype/Tests/dt_KeisterFun.py
"""
>>> from KeisterFun import KeisterFun as KeisterFun
>>> kf = KeisterFun()
>>> kf.dimension
2
>>> kf.domainType
'box'
>>> kf.distrib['name']
'IIDZGaussian'
>>> kf.nominalValue
0
>>> import numpy as np
>>> kf.g(np.array([[1, 2], [3, 4]]), [1, 2])
array([-1.93921993,  0.89115104]) 
"""

File: deprecated/MATLAB_Prototype/Tests/dt_KeisterFun.m
%% dt_KeisterFun
% doctest for KeisterFun
%
%  Examples
%
%  Example 1: Create a KisterFun instance.
%
% >> kf = KeisterFun
%
%   kf = 
%
%    KeisterFun with properties:
%     
%     domain: [2x2 double]
%     domainType: 'box'
%     dimension: 2
%     distribType: 'uniform'
%     nominalValue: 0    
%
%
%  Example 2: Invoke method f on the KisterFun instance.
%
%  >> y = kf.f([1,2; 3 4], [1 2])
% 
%   y =
%
%    -0.0042
%     0.0000
%
%
